pkgs/by-name/tl/tldx/package.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub, nix-update-script }:
+
+buildGoModule (finalAttrs: rec {
+  pname = "tldx";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "brandonyoungdev";
+    repo = pname;
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-MEFj8GaHXoDo2rAP+tOnisgLSZDqleh4pzX7M4CZBUk=";
```

Use hash for compatibility with other hash formats

```suggestion
    hash = "sha256-MEFj8GaHXoDo2rAP+tOnisgLSZDqleh4pzX7M4CZBUk=";
```